BraveRewardsUI/Wallet/WalletViewController.swift

```diff
@@ -171,9 +171,13 @@ class WalletViewController: UIViewController, RewardsSummaryProtocol {
       guard let host = state.url.host else { return }
       attentionView.valueLabel.text = "0%"
       
-      state.ledger.publisherInfo(forId: host) { info in
-        guard let publisher = info else { return }
+      state.ledger.publisherInfo(forId: host) { [weak self] info in
+        guard let self = self else { return }
+        
         assert(Thread.isMainThread)
+        publisherView.checkAgainButton.isHidden = info != nil
```

Comment:
This will also hide `check again` button if publisher is in database but not verified, right?
we can use `isHidden == publisher.isVerified`(after line 180)